code/datums/abilities/wizard/animal.dm

```diff
@@ -46,6 +46,8 @@ var/list/animal_spell_critter_paths = list(/mob/living/critter/small_animal/cat,
 	voice_fem = "sound/voice/wizard/FurryFem.ogg"
 	voice_other = "sound/voice/wizard/FurryLoud.ogg"
 
+	var/maptext_colors = list("#167935", "#9eee80", "#ee59e3", "#5a1d8a", "#ee59e3", "#9eee80")
```

Comment:
Should also define `maptext_colors` on the base `/datum/targetable/spell`.

Reply:
done! makes sense